BookingPages/LanguagePage.cs

```diff
@@ -0,0 +1,31 @@
+﻿using OpenQA.Selenium;
+using Utilities;
+
+namespace BookingPages
+{
+    public class LanguagePage
+    {
+        private readonly IWebDriver _driver;
+        private By LanguageElement => By.XPath("//*[text()='Nederlands']");
```

Comment:
does always text is Nederlands? what if in another test will be another country?

Reply:
Changed